modules/claims_api/app/sidekiq/claims_api/poa_vbms_updater.rb

```diff
@@ -54,15 +56,15 @@ def update_poa_access(poa_form:, participant_id:, poa_code:, allow_poa_access:)
         response = service.update_poa_access(
           participant_id:,
           poa_code:,
-          allow_poa_access:,
+          allow_poa_access: allow_poa_access ? 'Y' : 'N',
           allow_poa_c_add: allow_address_change?(poa_form) ? 'Y' : 'N'
         )
       else
         service = bgs_ext_service
         response = service.corporate_update.update_poa_access(
```

Comment:
we could just define the service in the flipper feature and make the call standardized so it's less code, so something like

```
    def update_poa_access(poa_form:, participant_id:, poa_code:, allow_poa_access:)
      # allow_poa_c_add reports 'No Data' if sent lowercase
      if Flipper.enabled? :claims_api_poa_vbms_updater_uses_local_bgs
        service = corporate_update_service
      else
        service = bgs_ext_service.corporate_update
      end
      service.update_poa_access(
        participant_id:,
        poa_code:,
        allow_poa_access: allow_poa_access ? 'Y' : 'N',
        allow_poa_c_add: allow_address_change?(poa_form) ? 'Y' : 'N'
      )
    end
    ```

Reply:
[3f8598a](https://github.com/department-of-veterans-affairs/vets-api/pull/19792/commits/3f8598ac12737935569a2560139aaf8b04e49f76)